hub/core/tensor.py

```diff
@@ -101,11 +107,18 @@ def add_samples_to_tensor(
         else:
             # TODO: we may want to call `tobytes` on `array` and call memoryview on that. this may depend on the access patterns we
             # choose to optimize for.
-            b = memoryview(tobytes(sample))
-
+            b = tobytes(sample)
+            if not tensor_meta.get("is_compressed", False):
```

Comment:
Can this be simplified. Also I think this would need to be updated after meta class PR merge.